README.rst

```diff
@@ -147,17 +147,20 @@ The PyAbel code adheres to the following conventions:
         THETA = np.arctan2(X, Y)
 
     where the usual ``(Y, X)`` convention of ``arctan2`` has been reversed in order to place zero angle in the vertical direction. Consequently, to convert the angular grid back to the Cartesian grid, we use
-  
+
     .. code-block:: python
 
         X = R * np.sin(THETA)
         Y = R * np.cos(THETA)
 
     The ``abel.tools.polar.cart2polar`` and ``abel.tools.polar.polar2cart`` functions are available for conversion between these Cartesian and polar grids.
 
-- 
+-
     **Image origin:** Fundamentally, the forward and inverse Abel transforms in PyAbel consider the origin of the image to be located in the center of a pixel. This means that, for a symmetric image, the image will have a width that is an odd number of pixels. (The central pixel is effectively "shared" between both halves of the image.) In most situations, the image origin is specified using the ``origin`` keyword in ``abel.Transform`` (or directly using ``abel.center.center_image`` to find the origin (the center of symmetry) of your image). This processing step takes care of shifting the origin of the image to the middle of the central pixel. However, if the individual Abel transforms methods are used directly, care must be taken to supply a properly centered image. Some methods also provide low-level functions for transforming only the right half of the image (with the origin located in the middle of a 0th-column pixel).
 
+-
+    **Intensity:** Absolute pixel intensities can be arbitrary (withing the floating-point range), but the intensity scale must be linear. Keep in mind that cameras and common image formats can use `gamma correction <https://en.wikipedia.org/wiki/Gamma_correction>`_ and thus provide data with nonlinear intensity encoding. When working with such data, make sure to apply appropriate intensity transformations before the analysis. Most PyAbel methods also assume intensities to be floating-point numbers, and when applied to integer types, can return inappropriately rounded results. The ``abel.Transform`` class recasts the input image to ``float64`` by default, but if you wish to call the transform methods directly or use other tools, you might need to perform the conversion yourself (as ``IM.astype(float)``, for example).
```

Comment:
Bold indicates suggested changes:

"**The pixel intensities can have any value (within the floating-point range). However, the intensity scale must be linear.**"

"...cameras and common image formats **often** use" 

"...nonlinear intensity encoding. **Thus, if possible, it is recommended to disable the gamma correction on cameras used to record images that will be inverse Abel transformed. If this is not possible, then it is necessary to apply the appropriate intensity transformations before the analysis.**"

Also, what rounding issues arise with the int-to-float conversion?

Reply:
Thanks!